lib/catindexinfo.go

```diff
@@ -61,7 +64,8 @@ func NewCatIndexInfo(indexLine string) (catIndex *CatIndexInfo, err error) {
 // Pull all the index info from the connection
 func (c *Conn) GetCatIndexInfo(pattern string) (catIndices []CatIndexInfo) {
 	catIndices = make([]CatIndexInfo, 0)
-	args := map[string]interface{}{"bytes": "b"}
+	//force it to only show the fileds we know about
+	args := map[string]interface{}{"bytes": "b", "h": "health,status,index,pri,rep,docs.count,docs.deleted,store.size,pri.store.size"}
```

Comment:
this should fix the backwards compatibility issue and also make it harder to break in future versions where they add new fields.